shell/platform/tizen/flutter_tizen_engine.cc

```diff
@@ -129,9 +129,9 @@ bool FlutterTizenEngine::RunEngine() {
       switches.begin(), switches.end(), std::back_inserter(argv),
       [](const std::string& arg) -> const char* { return arg.c_str(); });
 
-  if (std::find(switches.begin(), switches.end(), "verbose-logging") !=
+  if (std::find(switches.begin(), switches.end(), "--verbose-logging") !=
```

Comment:
This change is already included in https://github.com/flutter-tizen/engine/pull/142/files#